src/skins/blau.tsx

```diff
@@ -1,294 +1,296 @@
-import {applyAlpha} from '../utils/color';
 import {BLAU_SKIN} from './constants';
+import {applyAlpha} from '../utils/color';
 
-import type {GetKnownSkin} from './types';
-
-// https://github.com/Telefonica/mistica-design/blob/production/tokens/blau-constants.json
```

Comment:
tokens files don't use '-constants' in their names
https://github.com/Telefonica/mistica-design/blob/production/tokens/blau.json